internal/cmd/cmd_expiretime.go

```diff
@@ -0,0 +1,43 @@
+package cmd
+
+import (
+	dstore "github.com/dicedb/dice/internal/store"
+	"github.com/dicedb/dicedb-go/wire"
+)
+var cExpireTime = &DiceDBCommand{
+	Name: "EXPIRETIME",
+	HelpShort: `EXPIRETIME returns the absolute Unix timestamp (since January 1, 1970) in seconds
+	at which the given key will expire`,
+	Eval: evalExpireTime,
+}
+func init() {
+	commandRegistry.AddCommand(cExpireTime)
+}
```

Comment:
Run linter. This seems to be a listing miss. There should have been a space between functions and comments and globals.

Reply:
I ran the linter and made changes, please check if I am still missing anything.